qea7-Metal/src/test/java/foundation/TestBase.java

```diff
@@ -0,0 +1,40 @@
+package foundation;
+
+import java.net.URL;
+
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+
+public abstract class TestBase {
+
+	private ChromeDriver driver;
+
+	@BeforeMethod
+	public void launchDriver() {
+		URL driverPathForWindows = getClass().getResource("windows/chromedriver.exe");
```

Comment:
Move this into the if statement. No sense in trying to get a windows resource if we won't use it.